### DISCUSSION TITLE
How to get involved with contributing to Mantine's upcoming gallery.

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [mantinedev/mantine#515] [Q&A] [answered]
There have been talks about a new Mantine gallery that would probably come out in Q1 of 2022 (And for those who have contributed, can also see these by running `npm run storybook:gallery`). Is there any chance that information could be shared on what components are in the works (or what are you looking for in terms of different types of components like Ecommerce, Application, Marketing based), so that those who wish to contribute these components can do so ? This would also allow a mantine user to get more familiar with different components & available props API.

Is there perhaps any interest in this ?

**Comment by rtivital** [comment, marked as answer]
The gallery is in early stages of development. To contribute, you can follow this guide:
- Start gallery locally `npm run gallery`
- Choose category that you want to contribute to from the list
- Propose component that should be added to gallery in “Contributing” channel in Discord or via GitHub discussions, include following information: 
  - Component category
  - Design reference (you can find designs references on Dribble/Behance)
  - Make sure that component is not listed in documentation on mantine.dev
- If component is approved, build it and submit a pull request
- Make sure that component works well on gallery website and storybook

**Reply by UltraInstinct05** [reply]
> Choose category that you want to contribute to from the list

What list are you referring to ? Is there a public TODO list or something (which is surprising as I follow Mantine's development quite closely and never heard of it before) ?

**Reply by rtivital** [reply]
You can find categories list by launching `npm run gallery`, it is located on home page
